store/localcachelayer/channel_layer.go

```diff
@@ -70,6 +81,15 @@ func (s LocalCacheChannelStore) InvalidateGuestCount(channelId string) {
 	}
 }
 
+func (s LocalCacheChannelStore) InvalidateAllChannelMembersForUser(userId string) {
```

Comment:
Because this is public and is doing the cache invalidation in the whole cluster, you have to update the code around the `InvalidateAllChannelMembersForUser` calls that send the invalidate cluster messages.